src/schematools/contrib/django/management/commands/change_dataset.py

```diff
@@ -68,14 +72,22 @@ def handle(self, *args, **options):
             available = ", ".join(sorted(Dataset.objects.values_list("name", flat=True)))
             raise CommandError(f"Dataset not found: {name}.\nAvailable are: {available}") from None
 
-        # Validate illogical combinations
-        if options.get("endpoint_url"):
+        # Validate endpoint_url.
+        endpoint_url = options.get("endpoint_url")
+        if endpoint_url is not None:
             if options.get("enable_db"):
                 raise CommandError("Can't use --endpoint-url with --enable-db")
 
             if dataset.enable_db:
                 options["enable_db"] = False
 
+            if not endpoint_url.endswith("/"):
+                raise CommandError("--endpoint-url argument must have a trailing slash")
```

Comment:
Dit kan je niet verplichten van remote API's. Ze mogen ook eindigen met `{table_id}`.

Reply:
In dat geval wordt de table_id verwijderd door urljoin:

```python
url = "foo/{table_id}"
url = url.replace("{table_id}", "bar")
url = urljoin(url, "123")
```

`url` is nu `"foo/123"` ipv. `"foo/bar/123"`.

Reply:
Misschien moeten we more_ds hier gebruiken in plaats van urljoin?